client/dive-common/components/AttributeEditor.vue

```diff
@@ -128,14 +162,71 @@ export default defineComponent({
             required
           />
           <v-select
-            v-model="datatype"
+            :value="datatype"
             :items="[
               { text: 'Boolean', value: 'boolean' },
               { text: 'Number', value: 'number' },
               { text: 'Text', value: 'text' }
             ]"
             label="Datatype"
+            @change="typeChange"
           />
+          <div v-if="datatype=== 'number'">
+            <v-radio-group
+              :value="editor.type"
```

Comment:
I think this line is causing an error for existing attributes. When you click the edit attribute cog the dialog fails to load and console displays this:
![image](https://user-images.githubusercontent.com/40494088/184516439-12e48a9c-04cf-4a44-97c4-d8e732ea748a.png)

Reply:
Yup, editor probably needs to be checked to see if it exists, editor can be undefined.